src/OpenTelemetry.Extensions.Enrichment/Internal/TraceEnrichmentActions.cs

```diff
@@ -21,21 +21,21 @@
 namespace OpenTelemetry.Extensions.Enrichment;
 
 #pragma warning disable CA1812 // Class is instantiated through dependency injection
-internal sealed class EnrichmentActions : TraceEnricher
+internal sealed class TraceEnrichmentActions : TraceEnricher
 #pragma warning restore CA1812 // Class is instantiated through dependency injection
 {
     private readonly Action<TraceEnrichmentBag>[] actions;
 
-    public EnrichmentActions(IEnumerable<Action<TraceEnrichmentBag>> actions)
+    public TraceEnrichmentActions(IEnumerable<Action<TraceEnrichmentBag>> actions)
     {
         this.actions = actions.ToArray();
     }
 
-    public override void Enrich(in TraceEnrichmentBag enrichmentBag)
+    public override void Enrich(in TraceEnrichmentBag activity)
```

Comment:
Name of the parameter should match the interface signature - do we have this analysis rule disabled?

Reply:
I guess it must be as I didn't see any warnings regarding this. Fixed, thanks!